better_profanity/better_profanity.py

```diff
@@ -11,6 +13,9 @@
     get_complete_path_of_file,
 )
 
+# Maximum number of patterns that can be generated for a word.
+MAX_PATTERNS = 4000000
```

Comment:
Yes, I think this should be moved to `constants.py`.

May I ask how do you come up with this number ?

Reply:
From my tests in #15, this seems to be roughly where memory growth begins exploding (between 2 million and 4 million variants). I tried also `2000000`, `4000000`, and `5000000` on a word list of (875 demonetizable words and phrases for YouTube)[https://docs.google.com/spreadsheets/d/1ozg1Cnm6SdtM4M5rATkANAi07xAzYWaKL7HKxyvoHzk/edit#gid=674179785] and `4000000` was about as high as I could go before surpassing 2 GB. But even so, that 2 GB threshold is a somewhat arbitrary decision.

Reply:
I think we should lower that number. As 4.000.000 is the max number of patterns for 1 word, if we have multiple words with similar number of patterns, it would consume all the memory of the machine.

I think the `MAX_PATTERNS` should be around 5000. What do you think ?

Reply:
What is your opinion about having the `MAX_PATTERN` as an attribute in `Profanity` class ? By setting that, users could change the value if they want to.

Reply:
I would set it a bit higher than 5000. He's a sample of words from the default word list and their number of variants:
```
arse 36
autoerotic 9216
beastiality 18432
booooooobs 49152
```

Reply:
I see. Maybe set it as 20,000 or 30,000 ?